sw/device/tests/flash_ctrl_mem_protection_test.c

```diff
@@ -196,10 +197,13 @@ bool test_main(void) {
       &flash, mmio_region_from_addr(TOP_EARLGREY_FLASH_CTRL_CORE_BASE_ADDR)));
 
   // Set up default access for data partitions.
-  CHECK_STATUS_OK(flash_ctrl_testutils_default_region_access(
-      &flash, /*rd_en=*/true, /*prog_en=*/true, /*erase_en=*/true,
-      /*scramble_en=*/false, /*ecc_en=*/false, /*high_endurance_en=*/false));
-
+  // In silicon, rom_ext will set default region access.
+  // After that, it cannot be updated.
+  if (kBootStage != kBootStageOwner) {
```

Comment:
Technically, this is true of any execution environment using a ROM_EXT that locks things down, but since we aren't checking against `kDeviceType` here, we only have a misleading comment.